src/comm/aca_comm_mgr.cpp

```diff
@@ -80,7 +78,10 @@ int Aca_Comm_Manager::update_goal_state(GoalState &goal_state_message,
 
   ACA_LOG_DEBUG("Starting to update goal state\n");
 
-  ACA_LOG_INFO("Goal state message size is: %lu bytes\n", goal_state_message.ByteSizeLong());
+  ACA_LOG_INFO("!!!Goal state message size is: %lu bytes!!!\n",
```

Comment:
It seems that you really want people to see this message with so many "!" 

One recommendation: use something like "[METRICS]" to distinguish the message out.

Reply:
Updated, I like the idea.